src/js/components/Widgets/BallotItemSupportOpposeCountDisplay.jsx

```diff
@@ -629,7 +629,7 @@ class BallotItemSupportOpposeCountDisplay extends Component {
                   { totalNetworkScoreWithSign }
                 </span>
               )}
-              <NetworkScoreLabel className="u-show-mobile-tablet">Network Score</NetworkScoreLabel>
+              <NetworkScoreLabel className="u-show-mobile">Network Score</NetworkScoreLabel>
```

Comment:
We can actually get rid of the "Network Score" text. We are just using the number in the "score square".

Reply:
You mentioned that, thanks I'll remove it